Homework9/Hw9/Services/ExpressionCalculator/ExpressionMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Linq.Expressions;
+
+namespace Hw9.Services.ExpressionCalculator
+{
+    public class ExpressionMapper: ExpressionVisitor
+    {
+        private readonly Dictionary<Expression, List<Expression>> _expressionsMap = new();
+
+        public Dictionary<Expression, List<Expression>> MapExpression(Expression expression)
```

Comment:
Или что-то в этом роде
```suggestion
        public Dictionary<Expression, List<Expression>> ConstructExecuteBeforeMap(Expression expression)
```